ch05 ch06 rewrite for micro:bit v2 (#381)
rust-embedded/discovery: src/05-led-roulette/flash-it.md

By this I mean that there's nothing else running on the microcontroller: no OS, no "daemon",
nothing. `led-roulette` has full control over the device.

> Flashing the binary itself is quite simple thanks to `cargo-embed`.

Reviewer: ```suggestion
Flashing the binary itself is quite simple thanks to `cargo embed`.
```

Before executing that command though, lets look into what it actually does. If you look at the side of your micro:bit
with the USB connector facing upwards you will notice, that there are actually 2 black squares on there

> (on the micro:bit v2 is a third and biggest one, its a speaker), one is our MCU

Reviewer: ```suggestion
(on the micro:bit v2 there is a third and biggest one, which is a speaker), one is our MCU
```
I am not sure about "biggest" here.

  (...)
     Erasing sectors ✔ [00:00:00] [####################################################################################################################################################]  2.00KiB/ 2.00KiB @  4.21KiB/s (eta 0s )
 Programming pages   ✔ [00:00:00] [####################################################################################################################################################]  2.00KiB/ 2.00KiB @  2.71KiB/s (eta 0s )
    Finished flashing in 0.608s

# For micro:bit v1
$ cargo embed --features v1 --target thumbv6m-none-eabi
  (...)
     Erasing sectors ✔ [00:00:00] [####################################################################################################################################################]  2.00KiB/ 2.00KiB @  4.14KiB/s (eta 0s )
 Programming pages   ✔ [00:00:00] [####################################################################################################################################################]  2.00KiB/ 2.00KiB @  2.69KiB/s (eta 0s )
    Finished flashing in 0.614s
```

You will notice that `cargo-embed` blocks after outputting the last line, this is inteded and you should not close it

> since we need it in this state for the next step, debugging it! Furthermore you will have noticed that the `cargo build`

Reviewer: ```suggestion
since we need it in this state for the next step: debugging it! Furthermore, you will have noticed that the `cargo build`
```
